Lesson 1/ppl_ages_parser.py

```diff
@@ -0,0 +1,45 @@
+import json
+import yaml
+import urllib.request
+
+
+json_source = urllib.request.urlopen("https://raw.githubusercontent.com/IdoBaram/opsschool4-coding/master/hw.json")
+json_raw = json_source.read()
+j = json.loads(json_raw .decode("utf-8"))
```

Comment:
Try to think of a meaningful name.
You could also do:
`json_raw = json_source.read().loads(json_raw .decode("utf-8"))`
in one line